compiler/rustc_pattern_analysis/src/usefulness.rs

```diff
@@ -949,14 +945,13 @@ impl<'p, Cx: TypeCx> MatrixRow<'p, Cx> {
     /// Only call if `ctor.is_covered_by(self.head().ctor())` is true.
     fn pop_head_constructor(
         &self,
-        pcx: &PlaceCtxt<'_, 'p, Cx>,
         ctor: &Constructor<Cx>,
-        ctor_sub_tys: &[Cx::Ty],
+        ctor_arity: usize,
```

Comment:
I guess if we had kept pcx around, we could've called `pcx.arity(ctor)`, but it's easier to just pass it in here rather than recompute it?

Reply:
Yeah, computing it can take a few steps for cases like structs where we have to skip private uninhabited fields